aio/content/examples/router-tutorial/src/app/pagenotfound/pagenotfound.component.html

```diff
@@ -0,0 +1,2 @@
+<h2>Page Not Found</h2>
```

Comment:
```suggestion
<h3>Page Not Found</h3>
```
Oh btw, don't forget to adjust that for the e2e test (if you're using the one from slack today) because that one references the h2.

Reply:
I changed the test to look for the second h2 element, which would be the Page Not Found element. That should do the trick, right?